packages/dds-components/src/components/ToggleButton/ToggleButton.types.tsx

```diff
@@ -11,6 +11,8 @@ export type ToggleButtonProps = BaseComponentProps<
     label?: string;
     /** Ikon. */
     icon?: SvgIcon;
+    /**Størrelse. */
```

Comment:
Trenger default her for Storybook docs:
```suggestion
    /** Størrelse. 
     * @default "small"
     */
```